### PR TITLE
fix: add cert manager resources

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

### What does this PR do?
Add missing cert manager resources

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/20778

### How to test this PR?

See issue

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: bin/p.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+spec:
```
Comment (AndrienkoAleksandr) [inline bin/p.yaml:1]: Sorry, do we need this file?

File: src/commands/server/deploy.ts [hunk 2]
```diff
@@ -345,12 +345,6 @@ export default class Deploy extends Command {
       }
     }
 
-    if (!flags.batch && isKubernetesPlatformFamily(flags.platform) && (isDevWorkspaceEnabled(ctx) || flags['workspace-engine'] === 'dev-workspace')) {
```
Comment (mmorhun) [inline src/commands/server/deploy.ts:348]: How this is related to adding resources for Cert Manager?

File: bin/p.yaml (hunk 1 above)
Comment (tolusha) [reply bin/p.yaml]: fixed

File: src/commands/server/deploy.ts (hunk 2 above)
Comment (tolusha) [reply src/commands/server/deploy.ts:348]: Removing code duplication. Just spotted accidentally.

Comment (tolusha) [reply src/commands/server/deploy.ts:348]: https://github.com/che-incubator/chectl/blob/main/src/commands/server/deploy.ts#L342-L353

**Review by mmorhun — APPROVED** [review]

**Pull request merged**